acceptance/indy_acceptance/indy-tests/utils/constant.py

```diff
@@ -0,0 +1,44 @@
+'''
+Created on Nov 9, 2017
+
+@author: khoi.ngo
+'''
+
+
+class Colors:
+    """ 
+    Class to set the colors for text.
+    Syntax:  print(Colors.OKGREEN +"TEXT HERE" +Colors.ENDC)
+    """
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'  # Normal default color
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+class Roles:
+    """ 
+    Class to define roles.
+    """
+    TRUSTEE = "TRUSTEE"
+    STEWARD = "STEWARD"
+    TRUST_ANCHOR = "TRUST_ANCHOR"
+    TGB = "TGB"  # obsolete
+    NONE = ""
+
+
+class Constant:
+    """ 
+    Class Constant store some necessary paths.
+    """
+    import os
+    work_dir = os.path.expanduser('~') + os.sep + ".indy"
```

Comment:
The current working dir for libindy is .indy_client, not .indy (in the latest stable)

Reply:
When we run tests with libindy, the .indy directory is created automatically.  I do not believe .indy_client is the default directory anymore.

Reply:
What version of libindy are you using?
I think it was changed in the latest stable (which was issued last week)

Reply:
I am using the version from the stable build dated 01-Nov-2017.  That is the only build I see in the 1.0.1 directory